packages/suite/src/views/onboarding/steps/ResetDevice.tsx

```diff
@@ -1,111 +1,154 @@
-import { useState } from 'react';
-
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
 import { selectDevice } from '@suite-common/wallet-core';
-
 import * as STEP from 'src/constants/onboarding/steps';
-import {
-    OnboardingButtonBack,
-    OnboardingOption,
-    OptionsWrapper,
-    OptionsDivider,
-    OnboardingStepBox,
-} from 'src/components/onboarding';
+import { OnboardingButtonBack, OptionsWrapper, OnboardingStepBox } from 'src/components/onboarding';
 import { Translation } from 'src/components/suite';
 import { useDispatch, useSelector, useOnboarding, useDevice } from 'src/hooks/suite';
 import { resetDevice } from 'src/actions/settings/deviceSettingsActions';
 import { selectIsActionAbortable } from 'src/reducers/suite/suiteReducer';
+import { Button, Divider, Text } from '@trezor/components';
+import {
+    BackupType,
+    SelectBackupType,
+    defaultBackupTypeMap,
+    isShamirBackupType,
+} from './SelectBackupType';
+import { DeviceModelInternal } from '@trezor/connect';
+
+const SelectWrapper = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const ButtonWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+`;
+
+const canChooseBackupType = (device: DeviceModelInternal) => device !== DeviceModelInternal.T1B1;
 
 export const ResetDeviceStep = () => {
     const { isLocked } = useDevice();
+    const device = useSelector(selectDevice);
+    const isActionAbortable = useSelector(selectIsActionAbortable);
+
+    const device_model = device?.features?.internal_model;
```

Comment:
Is there any good reason why to use a snake_case for this variable?

Reply:
I have no idea, its just there. I assume its something that comes from firmware api, but its just guess.

Reply:
I mean not the `device?.features?.internal_model` but the `const device_model`. Snake case variables are used in e. g. Python not TypeScript.

I would use `const deviceModel`

Reply:
ah I see :D sorry I am blind! True! Fixed :white_check_mark: